model/countries/AU/AU-metadata.yaml

```diff
@@ -1,2 +1,6 @@
 country: AU
-flag: 🇦🇺
\ No newline at end of file
+flag: 🇦🇺
+
+
+overview: |
+    Addresses in Australia typically follow the structure described in the following <a href="https://auspost.com.au/content/dam/auspost_corp/media/documents/australia-post-addressing-standards-1999.pdf" > DOCUMENT </a>
```

Comment:
Delete space after DOCUMENT and add a period.

`DOCUMENT</a>.`

Reply:
Done